core-kotlin-modules/core-kotlin-10/src/test/kotlin/com/baeldung/variableshadowing/VariableShadowingUnitTest.kt

```diff
@@ -0,0 +1,188 @@
+@file:Suppress("UNUSED_VARIABLE", "UNUSED_ANONYMOUS_PARAMETER")
+
+package com.baeldung.variableshadowing
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class VariableShadowingUnitTest{
+
+    @Test
+    fun `Top-level variable shadowing`(){
```

Comment:
Since every other test name is lower case
```suggestion
    fun `top-level variable shadowing`(){
```